compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -74,6 +76,19 @@ template <> void geneate_random_data<bool>(std::mt19937 &gen, void *data, uint32
   }
 }
 
+template <typename T>
+void geneate_random_range(void *data, uint32_t size, int32_t range_min, int32_t range_max)
+{
+  for (uint32_t i = 0; i < size; i++)
+  {
+    // +1 will make value of [range_min, range_max]
+    int32_t range = range_max - range_min + 1;
+    int32_t value = (rand() % range) + range_min;
```

Comment:
Not a problem but just for your information :D

---

Range of `rand()` function is [0, RAND_MAX].
One interesting thing is that RAND_MAX is different between Windows and Linux.

- Windows RAND_MAX =0x7fff = 32767 = 2^15-1
- Linux RAND_MAX = 0x7fffffff = 2147483647 = 2^31-1

So, when I call `generate_random_range(data, size, 0, 40000)` in Windows, I would get following random values.
- Expectation : [0, 40000]
- Actual : [0,32768]

As actual range is included in expectation range, it is not incorrect, just different with our expectation :D